src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
ugyes

Reply:
javítsd kérlek ezt szebbé

Reply:
Thanks for the review, i will make suggested changes.

Reply:
Sorry but your suggested change is a bullshit.